scripts/stanford-filter.script

```diff
@@ -57,6 +57,33 @@
      )
    )
   )
+   
+  // GE SIGNA
+  +( (Manufacturer.startsWithIgnoreCase("GE MEDICAL") + Manufacturer.startsWithIgnoreCase("GEMS"))
+    * (Modality.equalsIgnoreCase("PT") + Modality.equals("OT") + Modality.equals("CT"))
```

Comment:
Is this too broad? I tried keeping only PT and remove OT and CT, it filters out few series